packages/core-mobile/app/screens/bridge/hooks/usePendingBridgeTransactions.ts

```diff
@@ -1,47 +1,13 @@
-import { BridgeTransaction } from '@avalabs/core-bridge-sdk'
 import { BridgeTransfer } from '@avalabs/bridge-unified'
 import { Network } from '@avalabs/core-chains-sdk'
 import { useMemo } from 'react'
 import { useSelector } from 'react-redux'
-import { isAvalancheNetwork } from 'services/network/utils/isAvalancheNetwork'
-import { isEthereumNetwork } from 'services/network/utils/isEthereumNetwork'
-import { selectBridgeTransactions } from 'store/bridge'
 import { selectPendingTransfers } from 'store/unifiedBridge/slice'
 import { caipToChainId } from 'utils/data/caip'
-import { isBitcoinNetwork } from 'utils/network/isBitcoinNetwork'
 
-const usePendingLegacyBridgeTransactions = (
-  network?: Network
-): BridgeTransaction[] => {
-  const pendingBridgeByTxId = useSelector(selectBridgeTransactions)
-
-  return useMemo(() => {
-    if (!network) {
-      return Object.values(pendingBridgeByTxId)
-    }
-
-    const networkNameToCheck = isBitcoinNetwork(network)
-      ? BridgeNetwork.BITCOIN
-      : isAvalancheNetwork(network)
-      ? BridgeNetwork.AVALANCHE
-      : isEthereumNetwork(network)
-      ? BridgeNetwork.ETHEREUM
-      : null
-
-    return [
-      ...Object.values(pendingBridgeByTxId).filter(
-        tx =>
-          (tx.sourceChain.valueOf() === networkNameToCheck ||
-            tx.targetChain.valueOf() === networkNameToCheck) &&
-          tx.environment === (network.isTestnet ? 'test' : 'main')
-      )
-    ]
-  }, [network, pendingBridgeByTxId])
-}
```

Comment:
how will it look like if a user has a pending legacy bridge transaction and they upgrade to this version?

Reply:
reverted codes for showing the pending legacy transactions. Let's remove it again when we deprecate our legacy bridging entirely.